openmmtools/utils.py

```diff
@@ -177,9 +180,12 @@ def math_eval(expression, variables=None):
                  ast.Pow: operator.pow, ast.USub: operator.neg}
 
     # Supported functions, not defined in math.
-    functions = {'step': lambda x: 1 * (x >= 0),
-                 'step_hm': lambda x: 0.5 * (np.sign(x) + 1),
-                 'sign': lambda x: np.sign(x)}
+    if functions is None:
+        functions = {}
+    functions.update({'step': lambda x: 1 * (x >= 0),
```

Comment:
Did you want to do the opposite?
```python
functions = {'step': ..., 'step_hm': ...}
if user_functions is not None:
    functions.update(user_functions)
```
or did you wanted the default functions to overwrite a custom `step` function?